packages/config/config/devices/0x027a/zse29.json

```diff
@@ -16,6 +16,97 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "PIR Control",
+			"maxNodes": 4
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Report Value Sent to Associated Devices",
+			"description": "Set the value of basic set report sent to the light associated with the sensor in Group 2 (so the light turns on to the selected brightness level). 99 is full brightness.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
```

Comment:
Basic Set also supports 255 for "on with last value". Is that possible with this device?

Reply:
I'm not sure.  This capability isn't mentioned in the Open Z-wave configuration file or a Smartthings device handler [https://github.com/krlaframboise/SmartThings/blob/master/devicetypes/krlaframboise/zooz-outdoor-motion-sensor-2-0.src/zooz-outdoor-motion-sensor-2-0.groovy ](url)
Both mention a range of 0-99 only.

Reply:
Okay lets stay on 0-99 then.